integration_tests/mockApis/auth.ts

```diff
@@ -61,7 +61,7 @@ const redirect = () =>
         'Content-Type': 'text/html',
         Location: 'http://localhost:3007/sign-in/callback?code=codexxxx&state=stateyyyy',
       },
-      body: '<html><body>SignIn page<h1>Sign in</h1></body></html>',
+      body: '<html lang="en"><head><title>Mock SignIn page</title></head><body><h1>Sign in</h1><span class="govuk-visually-hidden" id="pageId" data-qa="sign-in"></span></body></html>',
```

Comment:
I had to update the wiremock served stub pages for hmpps-auth (ie. sign in page) so that they have the bare minimum to pass accessibility tests